staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/types.go

```diff
@@ -67,5 +69,25 @@ type Key struct {
 	Secret string `json:"secret"`
 }
 
+// GKMSConfig contains API configuration for Google KMS transformer.
+type GKMSConfig struct {
+	// projectID is the GCP project which hosts the key to be used. It defaults to the GCP project
+	// in use, if you are running on Kubernetes on GKE/GCE. Setting this field will override
+	// the default. It is not optional if Kubernetes is not on GKE/GCE.
+	// +optional
+	ProjectID string `json:"projectID,omitempty"`
+	// location is the location of the KeyRing to be used for encryption. The default value is "global".
```

Comment:
In GCP location means region or zone. This should be stated more clearly. At least "GCP location" so someone could reasonably Google the term.